packages/audiowmark/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Nils Hilbricht <nils@hilbricht.net>
+
+pkgname=audiowmark
+pkgver=0.6.1
+pkgrel=1
+pkgdesc='Audio and video watermarking'
+arch=(x86_64 aarch64)
+url='https://uplex.de/audiowmark/'
+license=(GPL3)
+#depends=(fftw libsndfile libgcrypt zita-resampler mpg123 ffmpeg) #explicit from README
+depends=(zita-resampler fftw ffmpeg mpg123) #as namcap sees them
```

Comment:
When in doubt, follow `namacp`. at least where "normal" dependencies are concerned (don't trust it for Python package dependencies, though).